docs/chapters/workflow/git-workflow.rst

```diff
@@ -65,3 +65,26 @@ force-pushed to the forked repo, or they can be added on top of the existing com
 request will be updated either way.
 
 Whenever the team is satisfied with the changes, pull the changes in by accepting the pull request.
+
+Good commits
+============
+
+A good commit (or patch):
+
+#. is atomic: Each commit should contain only one logical change. Use of vague verbs (e.g "update")
+   and phrases like "this patch also" are usually a good indicator that commit is not atomic.
+
+#. has a good commit message associated with it.
```

Comment:
There needs to be an explanation or at least a link to an explanation of what we believe a good commit message is. Furthermore, a good commit message is a bit to vague. We have rules of which some are not trivial which contributors never will guess on their own.